lib/logitech_receiver/settings_templates.py

```diff
@@ -406,17 +400,6 @@ def __init__(self, device, dpiSetting):
                 we go into the `moved` state.
                 When the button is released in this state
                 the DPI is set according to the total displacement.
-
```

Comment:
Why remove this beauty of a diagram

Reply:
I'm in favour of brevity and the FSM is described in text.